src/index.ts

```diff
@@ -171,6 +177,10 @@ function startNewGame() {
   addTargetListener(currentTargets, COLORS);
 
   changeGameStatus(verifyButton, GameStatus.running);
+
+  console.log('Setting a new startDate');
+  runStart = new Date();
+  console.log(runStart);
 }
 
```

Comment:
A lot of console.log, interesting for development but we don't want to keep them for the main

Reply:
Done